in_toto/in_toto_record.py

```diff
@@ -128,6 +128,14 @@ def create_parser():
       " resulting link metadata's product section when running the 'stop'"
       " subcommand. Symlinks to files are followed."))
 
+  subparser_start.add_argument("-d", "--metadata-directory", type=str, required=False, 
+    metavar="<directory>", help=(
+      "specify where to write the generated metadata to."))
```

Comment:
FYI, there's an `in_toto.common_args` module that we use to define argparse arguments that are used multiple times only once. I think `--metadata-directory` qualifies for it. :)